src/google/protobuf/repeated_field.h

```diff
@@ -535,7 +535,7 @@ class ABSL_ATTRIBUTE_WARN_UNUSED RepeatedField final
   // Destroys all elements in [begin, end).
   // This function does nothing if `Element` is trivial.
   static void Destroy(const Element* begin, const Element* end) {
-    if (!std::is_trivial<Element>::value) {
+    if constexpr (!std::is_trivially_destructible<Element>::value) {
```

Comment:
Since now protobuf requires C++17, I added an `if constexpr`. Since this is only about calling a destructor, I guess it's ok to only check `is_trivially_destructible`.

Reply:
Agreed